modules/Users/Changenewpassword.php

```diff
@@ -191,5 +191,10 @@
     $sugar_smarty->assign("KEY", $_REQUEST['key']);
 }
 
-$sugar_smarty->display('modules/Users/Changenewpassword.tpl');
+if (file_exists('custom/modules/Users/Changenewpassword.tpl')) {
```

Comment:
I suggest replacing this entire block with
```php
$sugar_smarty->display(get_custom_file_if_exists(modules/Users/Changenewpassword.tpl));
```

Reply:
Thank you, better according to your proposal.
Changed